messagebird/client.py

```diff
@@ -193,9 +193,11 @@ def message(self, id):
         """Retrieve the information of a specific message."""
         return Message().load(self.request('messages/' + str(id)))
 
-    def message_list(self, limit=20, offset=0):
+    def message_list(self, limit=20, offset=0, status=None):
```

Comment:
would be helpful to know what the possible values for `status` are. Could you add that to the docstrings?

Reply:
good point! fixed